packages/terra-form/src/TextareaField.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Field from './Field';
+import Textarea from './Textarea';
+
+const propTypes = {
+  /**
+   * How many columns the textarea should have
+   */
+  cols: PropTypes.number,
+  /**
+   * The defaultValue of the textarea element
+   */
+  defaultValue: PropTypes.string,
+  /**
+   * Error message for when the input is invalid
+   */
+  error: PropTypes.node,
+  /**
+   *  Help element to display with the input
+   */
+  help: PropTypes.node,
+  /**
+   * Custom attributes to apply to the input
+   */
+  inputAttrs: PropTypes.object,
+  /**
+   * Whether the field is inline
+   */
+  isInline: PropTypes.bool,
+  /**
+   * Label of the input
+   */
+  label: PropTypes.node,
+  /**
+   * Maximum number of characters user can input in this field
+   */
+  maxLength: PropTypes.number,
+  /**
+   * Minimum number of characters user must input in this field
+   */
+  minLength: PropTypes.number,
+  /**
+   * Name of the input attribute
+   */
+  name: PropTypes.string,
+  /**
+   * Function to trigger when user selects one of the radio buttons
```

Comment:
I updated all the value, defaultValue, and onChange documentations here. https://github.com/cerner/terra-core/pull/410/commits/af45bfb9d7091fc8a1be4d4cc5f8836f71a125f8